reproschema/validate.py

```diff
@@ -57,13 +55,11 @@ def validate_dir(directory, shape_file, started=False, http_kwargs={}):
     return True
 
 
-def validate(shapefile, path):
+def validate(path):
```

Comment:
should this also check `schemaVersion` and either ask user to migrate or ask them to use the older shacl-based validator?

Reply:
but I thought that the idea was to use `fixing_old_schema` (or `migrate_schema`) for the older version. This is why I was checking entire `reproschema-library` with the new validation

Reply:
but this function doesn't check anything, right?

Reply:
it initialize the pydantic class, so will get an error if the arguments are off